lib/src/wallet/traits.rs

```diff
@@ -718,7 +716,7 @@ impl ReceivedNoteAndMetadata for ReceivedOrchardNoteAndMetadata {
     }
 }
 
-pub trait DomainWalletExt<P: Parameters>: Domain + BatchDomain
+pub trait DomainWalletExt: Domain + BatchDomain
```

Comment:
Why are we able to do this?   What suggested this change?  `clippy`?

Reply:
I remember trying and failing to do it in the past. I decided I wanted to try it again, and I don't know what I did wrong the first time, because it worked this time.